apis/config/v1alpha2/configuration_types.go

```diff
@@ -39,8 +39,27 @@ type Configuration struct {
 	// unsuspended, they will start immediately.
 	ManageJobsWithoutQueueName bool `json:"manageJobsWithoutQueueName"`
 
-	// EnableInternalCertManagement controls whether to enable internal cert management or not.
+	// InternalCertManagement is configuration for internalCertManagement
+	InternalCertManagement InternalCertManagement `json:"internalCertManagement,omitempty"`
```

Comment:
Make this a pointer, then we can know whether we set this or not.

Reply:
SGTM.

Reply:
Done.